src/main/java/com/jmb/springfactory/model/bo/QueryTaskObject.java

```diff
@@ -0,0 +1,21 @@
+package com.jmb.springfactory.model.bo;
+
+import java.util.Date;
+
+import com.jmb.springfactory.model.enumeration.PriorityEnum;
+import com.jmb.springfactory.model.enumeration.TaskStatusEnum;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class QueryTaskObject {
+
+  private String name;
+  private TaskStatusEnum status;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "status" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)